### DISCUSSION TITLE
org.eclipse.ui not find in eclipse rcp

### DISCUSSION THREAD

**Discussion by yangmeng66** [eclipse-platform/eclipse.platform.releng.aggregator#851] [Q&A] [answered]
Hello, what's the reason?

![image](https://user-images.githubusercontent.com/51857567/218031305-e254f264-87ae-4771-9844-ab2d63993918.png)
![image](https://user-images.githubusercontent.com/51857567/218031321-41c095f2-17ee-4819-a84b-2af6cc5704ad.png)

**Comment by merks** [comment]
Your image did not make it into the post.  You'll need to edit it...

It's there now.

**Reply by yangmeng66** [reply]
Sorry, can you see it now?

**Comment by akurtakov** [comment]
My first though would be that your "Target Platform" doesn't include it. You can check whether it's there by using "Target Platform State" view.

**Reply by yangmeng66** [reply]
Yes, it is not in the "Target Platform state". How to load this plug-in?

**Comment by merks** [comment, marked as answer]
What do the Target Platform Preferences look like?   I'd expect them to look exactly like this:

![image](https://user-images.githubusercontent.com/208716/218032521-7de7d3b1-f148-46e2-a5a2-9f0512b005d0.png)

Note the Reload... button...

**Reply by yangmeng66** [reply]
Forgive me for not understanding these settings of eclipse, thank you
![image](https://user-images.githubusercontent.com/51857567/218033885-f2ddeb0b-e824-4332-9957-b54a6668d68e.png)

**Comment by akurtakov** [comment]
Some links to read on the topic:
* https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.pde.doc.user%2Fconcepts%2Ftarget.htm
* https://www.vogella.com/tutorials/EclipseTargetPlatform/article.html

**Reply by yangmeng66** [reply]
Thank you very much. I will study hard

**Comment by merks** [comment]
It seems you've already created your own target platform as some point.  

![image](https://user-images.githubusercontent.com/208716/218035178-ff9e7aad-818a-4f17-bc40-746b5d548df3.png)

The Running Platform should have lots of available plugins as underlined above.

**Reply by yangmeng66** [reply]
Thank you very much.
